src/components/Balance.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import type {StyleProp, TextStyle} from 'react-native';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as CurrencyUtils from '@libs/CurrencyUtils';
+import Text from './Text';
+
+type BalanceProps = {
+    textStyles?: StyleProp<TextStyle>;
+    balance: number;
+};
+
+function Balance({textStyles, balance}: BalanceProps) {
+    const styles = useThemeStyles();
+    const formattedBalance = CurrencyUtils.convertToDisplayString(balance);
```

Comment:
NAB: I would add a whitespace between const and return just for readability ;)

Reply:
Done - f8e7928.